uniffi_bindgen/src/bindings/python/templates/RustBufferBuilder.py

```diff
@@ -1,6 +1,6 @@
 
 class RustBufferBuilder(object):
-    """Helper for structured writing of values into a RustBuffer."""
+    # Helper for structured writing of bytes into a RustBuffer.
```

Comment:
Any reason why you are changing this from a docstring to a standard comment?

Reply:
docstrings have a (trivial) runtime cost that I don't think is worth it here - this is never going to be rendered for a human - so while I was touching it.. :)